docs/SNAP_RECOMMENDATIONS.md

```diff
@@ -0,0 +1,146 @@
+
+There are three types of recommendations that Searchspring offers:
+
+- Product Recommendations / Personalized Recommendations
+- Bundle Recommendations
+- Email Recommendations (documentation coming soon)
+<!-- TODO: Add Email Recommendations Docs -->
+
+This guide will cover usage of the Default templates that are available in the Searchspring Management Console. If you are looking to create a custom template, please refer to the [Custom Templates reference](https://searchspring.github.io/snap/reference-custom-recommendation-templates)
+
+Changes to the recommendation integration scripts were made in Snap `v0.60.0`. Legacy Recommendation Integrations docs can still be found [`here`](https://searchspring.github.io/snap/snap-recommendations-legacy)
+
+
+## More Information
+See [Recommendations Controller reference](https://searchspring.github.io/snap/reference-controller-recommendation) for more information on all available context variables that can be used to configure recommendations. The reference also contains more information regarding batching, ordering, deduplication, filtering, examples, and more.
+
+## Product Recommendations / Personalized Recommendations
+
+The Searchspring Management Console contains a `Default` template available for standard profiles (non-bundle) that does not require the use of the Snapfu CLI to create a custom template. To use the `Default` template, the following instantiator config should be added to your `snap-preact` config.
+
+```typescript
+// src/index.ts
+
+const snap = new Snap({
+    client: {
+        globals: {
+            siteId: 'abc123',
+        },
+    },
+    instantiators: {
+		recommendation: {
+			components: {
+				Default: async () => {
+					return (await import('./components/Recommendations/Recs')).Recs;
+				},
+			},
+		},
+	},
+});
+```
+
+Note that the component is not required to be named `Default`, however `instantiators.recommendation.component` must contain the `Default` key as seen in the example above.
+
+
+This example assumes a `recently-viewed` profile has been configured in the Searchspring Management Console (SMC) with the `Default` template selected. The profile (specified via the `tag` property) will render inside the `.ss__recs__recently-viewed` element below the script block. While the target element can be placed anywhere on the page, it's recommended to group elements with their corresponding script blocks for easier integration management. The component configuration is handled within the [`RecommendationInstantiator`](https://searchspring.github.io/snap/reference-snap-preact-instantiators).
```

Comment:
I think maybe "The following example ..." might clarify we are not still talking about the above one.